src/app/checklists/items-list/item/item.component.scss

```diff
@@ -46,75 +51,72 @@
   color: white;
 }
 
-.item-warning .prompt {
-  color: yellow;
+.item-challenge .indentable-container:before,
+.item-challenge-response .indentable-container:before {
+  content: '□';
+  position: absolute;
+  top: 5px;
+  left: 0px;
+  background: #000;
+  width: 20px;
 }
 
-.item-note .prompt {
-  color: gray;
+.item-challenge .indentable-container,
+.item-challenge-response .indentable-container {
+  padding-left: 20px;
 }
 
-/* Expand editors of note and title to full available width */
-.item-text .prompt,
-.item-title .prompt,
-.item-text .prompt-input,
-.item-title .prompt-input {
-  width: 100%;
+.item-challenge-response .indentable-container {
+  background-image: url("data:image/svg+xml;charset=UTF-8,%3csvg width='20' height='20' viewBox='0 0 20 20' xmlns='http://www.w3.org/2000/svg'%3e %3Ccircle cx='10' cy='15' r='2' fill='white' /%3e %3c/svg%3e");
+  background-repeat: repeat-x;
+  background-position-y: 5px;
+  min-height: 1em;
 }
 
-.item-challenge .prompt:before,
-.item-challenge-response .prompt:before {
-  content: ' □';
-  vertical-align: 2px;
+.item-caution:not(.item-centered) .prompt:before,
+.item-warning:not(.item-centered) .prompt:before,
+.item-note:not(.item-centered) .prompt:before {
+  padding-right: 10px;
 }
 
 .item-caution .prompt:before {
   content: 'CAUTION:';
 }
 
+.item-warning .prompt {
+  color: yellow;
+}
+
 .item-warning .prompt:before {
   content: 'WARNING:';
 }
 
+.item-note .prompt {
+  color: gray;
+}
+
 .item-note .prompt:before {
   content: 'NOTE:';
 }
 
 .expectation-container {
-  flex-grow: 1;
   display: flex;
-  background-image: url("data:image/svg+xml;charset=UTF-8,%3csvg width='20' height='20' viewBox='0 0 20 20' xmlns='http://www.w3.org/2000/svg'%3e %3Ccircle cx='10' cy='15' r='2' fill='white' /%3e %3c/svg%3e");
-  background-repeat: repeat-x;
+}
+
+.expectation-spacer {
+  flex-grow: 1;
 }
 
 .expectation {
-  float: right;
   color: cyan;
   margin-top: 1px;
   padding-left: 5px;
   background-color: #000;
+  text-align: right;
 }
 
-.item-challenge .prompt-input,
-.item-challenge-response .prompt-input {
-  position: absolute;
-  left: 30px;
-  top: 1px;
-  background: #000;
-  padding-right: 5px;
-}
-
-.item-warning .prompt-input,
-.item-caution .prompt-input {
-  position: absolute;
-  left: 100px;
-}
-
-.expectation-input {
-  position: absolute;
-  right: 0;
+.prompt-input {
```

Comment:
Very minor/not blocking: you may want to add something like "padding-right: 5px" here to avoid the first dot being too close to the text (I tried adding the above in developer tools and it worked)

<img width="209" height="241" alt="Screenshot 2025-09-22 at 01 10 49" src="https://github.com/user-attachments/assets/73bacc5c-b337-4bc9-894a-116ef6ec637f" />